libimage/filters.go

```diff
@@ -275,7 +275,7 @@ func filterReferences(r *Runtime, wantedReferenceMatches, unwantedReferenceMatch
 		unwantedMatched := false
 		// Go through the unwanted matches first
 		for _, value := range unwantedReferenceMatches {
```

Comment:
Unwanted matches don’t affect the output names at all?

I’m not saying it’s obviously wrong, but I do think it is a design question that needs to be considered and explicitly decided.

Reply:
Okay, I'll put a `//TODO 6.0` note in there to remove unwanted matches from the output names.